pythran/pythonic/include/numpy/interp.hpp

```diff
@@ -13,38 +13,76 @@ namespace numpy
 
   // None,None,None
   template <class T1, class T2, class T3>
-  types::ndarray<double, types::pshape<long>>
+  typename std::enable_if<!std::is_arithmetic<T1>::value,
```

Comment:
do you mean std::is_integral here?

Reply:
ok, after discussing it on IRC, this change looks good, but doesn't capture `std::complex`. Is that intended?

Reply:
mmm no, I just didn't think of it. I should probably add a specialization for complex.